test/test_BC.sh

```diff
@@ -17,8 +17,12 @@ PREFIX=wt
 
 python $BIN_DIR/calc_network.py --topology $PREFIX.pdb --threshold 7.0 --step 100 --generate-plots --calc-BC --discard-graphs $PREFIX.dcd
 
-mv ${PREFIX}_0_bc.dat ref_${PREFIX}_bc.dat 
+mv ${PREFIX}_0_bc.dat ref_${PREFIX}_bc.dat
 
+python $BIN_DIR/calc_delta_BC.py --generate-plots --normalize --reference ref_${PREFIX}_bc.dat --alternatives ${PREFIX}_*_bc.dat --normalize
+python $BIN_DIR/calc_delta_BC.py --generate-plots --normalize --reference ref_${PREFIX}_bc.dat --alternatives ${PREFIX}_*_bc.dat --normalize --normalization-mode nonzero
+python $BIN_DIR/calc_delta_BC.py --generate-plots --normalize --reference ref_${PREFIX}_bc.dat --alternatives ${PREFIX}_*_bc.dat --normalize --normalization-mode standard
+python $BIN_DIR/calc_delta_BC.py --generate-plots --normalize --reference ref_${PREFIX}_bc.dat --alternatives ${PREFIX}_*_bc.dat --normalize --normalization-mode plusone
```

Comment:
Normalize flag being used twice in these commands. No test for the command without normalize flag set.